Our version of Mercurial doesn't have iteritems & itervalues available
for import; those were added in Mercurial 5.2.

This is Solaris specific and not suitable for upstream.

--- hg-git-0.10.4/hggit/compat.py
+++ hg-git-0.10.4/hggit/compat.py
@@ -80,9 +80,13 @@ except ImportError:
 try:
     from mercurial.pycompat import iteritems, itervalues
 except ImportError:
-    assert not pycompat.ispy3
-    iteritems = lambda x: x.iteritems()
-    itervalues = lambda x: x.itervalues()
+    # we have older version of Mercurial, which doesn't have these yet
+    if pycompat.ispy3:
+        iteritems = lambda x: x.items()
+        itervalues = lambda x: x.values()
+    else:
+        iteritems = lambda x: x.iteritems()
+        itervalues = lambda x: x.itervalues()
 
 try:
     # added in 5.9
